content/en/docs/instrumentation/net/getting-started.md

```diff
@@ -83,24 +78,127 @@ Resource associated with Activity:
     service.instance.id: 20c891c2-94b4-4203-a960-93a22e837a32
 ```
 
+</details>
+
 This output matches the span created in the preceding code sample.
 
+### Metrics
+
+Next, add metrics to the tracing code. This will initialize a
+[Meter](/docs/concepts/signals/metrics), and will use the Meter to create a
+Counter.
+
+```csharp
+using System.Diagnostics;
+using System.Diagnostics.Metrics;
+
+using OpenTelemetry;
+using OpenTelemetry.Trace;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Resources;
+
+// Define some important constants to initialize tracing with
+var serviceName = "MyCompany.MyProduct.MyService";
+var serviceVersion = "1.0.0";
+
+// Factor out the ResourceBuilder so it can be shared with the tracer provider
+// and the meter provider.
+var appResourceBuilder = 
+    ResourceBuilder.CreateDefault()
+        .AddService(serviceName: serviceName, serviceVersion: serviceVersion);
+
+var MyActivitySource = new ActivitySource(serviceName);
+
+// Configure important OpenTelemetry settings and the console exporter
+using var tracerProvider = Sdk.CreateTracerProviderBuilder()
+    .AddSource(serviceName)
+    .SetResourceBuilder(appResourceBuilder)
+    .AddConsoleExporter()
+    .Build();
+
+// Now configure Metrics
+var meter = new Meter("Example");
+var counter = meter.CreateCounter<long>("Example.Counter");
+var meterProvider = Sdk.CreateMeterProviderBuilder()
+    .AddMeter(meter.Name)
+    .AddConsoleExporter()
+    .Build();
+
+// Create a new span
+using var activity = MyActivitySource.StartActivity("SayHello");
+activity?.SetTag("foo", 1);
+activity?.SetTag("bar", "Hello, World!");
+activity?.SetTag("baz", new int[] { 1, 2, 3 });
+
+// Up the counter that was created
+counter.Add(1);
+
+// And flush the batch of metrics - for example only!
+meterProvider.ForceFlush()
```

Comment:
Is it intentional to leave out the `tracerProvider.ForceFlush`?

Reply:
It's not needed! The tracerprovider flushes the batch of traces on shutdown, but that doesn't appear to happen with the meterprovider. I'm not sure if that's a bug or not, but it's what seems to happen in my example app.